syntax = "proto3";
package Enze;
message Address
{
    uint64 nServices = 1;
    int32  ip = 2;
    int32  port = 3;
    uint32 nTime  = 4;
}
